theano/gpuarray/opt.py

```diff
@@ -55,14 +59,34 @@
 
 
 gpu_optimizer = EquilibriumDB()
+gpu_optimizer2 = EquilibriumDB()
```

Comment:
Add a comment saying we are not using it for an EquilibriumOptimizer, actually, but it has the "tracks" that we need for GraphToGPUDB.